### ISSUE TITLE
The latest smdebug in pypi is 1.0.12 and Released: Aug 26, 2021 

### ISSUE THREAD

**Issue by ZhangPei25** [awslabs/sagemaker-debugger#655]
When do pip install smdebug, the lastest version it can install is 1.0.12, and that is release 2 years ago.
But in github, the latest version is 1.0.31.
I wonder :
1. Is it still recommended to use smdebug in customer project or not, for example , use it in customer containers ? 
2. Why did not you publish new smdebug to pipy, any specific reasons ?

Thank you !

**Comment by pviolette3** [comment]
> Is it still recommended to use smdebug in customer project or not, for example , use it in customer containers ?

+1 - further, the docs here suggest the latest supported version is PyTorch 1.6, which was released nearly 3 years ago on [July 28, 2020](https://pytorch.org/blog/pytorch-1.6-released/).

Would be curious if anyone has figured out if more recent DL AMIs like PyTorch 1.13 (for instance) are supported.

**Comment by elgalu** [comment]
Exactly, I wonder if this package was silently deprecated, last release in 2021 :/
